functions/grapher/[slug].ts

```diff
@@ -153,8 +157,22 @@ async function handleHtmlPageRequest(
     // If we fail to capture the origin, we end up with relative image URLs, which should also be okay.
     let origin = ""
 
-    // Rewrite the two meta tags that are used for a social media preview image.
+    // Rewrite the fallback image (for no-JS cases) and meta tags that are used for a social media preview image.
     const rewriter = new HTMLRewriter()
+        .on("#grapher-preview-source", {
+            element: (element) => {
+                element.setAttribute(
+                    "srcset",
+                    generateGrapherImageSrcSet(thumbnailUrl)
+                )
+            },
+        })
+        // Grapher and data pages
+        .on(`img.GrapherImage`, {
+            element: (element) => {
+                element.setAttribute("src", thumbnailUrl)
+            },
+        })
```

Comment:
This is not quite the way to go. To see why, go to http://staging-site-remove-grapher-exports/grapher/child-mortality?time=1990 and scroll down to the related charts section.

I would suggest doing it like I did in https://github.com/owid/owid-grapher/pull/3873/files: There, I introduced a `[data-owid-populate-url-params]` attr, and only where that is present do we inject the page's query params.